### DISCUSSION TITLE
Audi A3 etron (2020) - Initalising Navigation System stuck at 11%

### DISCUSSION THREAD

**Discussion by williamsrh** [Mr-MIBonk/M.I.B._More-Incredible-Bash#525] [Q&A] [answered]
Hello all,
I bought the car just over a month ago and since then I've had an issue with the Navigation System not initialising, 
![Initialising navigation stuck at 11%](https://github.com/user-attachments/assets/3cfc155f-72bd-4ba0-aed0-2db0b9ea9cdd)
I obtained a copy of the M.I.B software and the newest 2024 map version in the hope that this would solve the issue, 
I went through the installation instructions for installing the M.I.B software and applying the activation codes all of which seem to have been done successfully, 
![Patch Unit Screen](https://github.com/user-attachments/assets/05e4e97b-2f8a-4e3e-8367-ac1fed3c42ec)
![Activation Codes](https://github.com/user-attachments/assets/812ef064-f608-46d5-975f-cf7c7c5e33ca)
However, I can't update the maps either through the online update in the car or using an SD card, I keep getting the error "No valid update data found on this device". 
![No valid update data found on the medium](https://github.com/user-attachments/assets/9ebadd06-7dbf-4b8c-b608-a1e58da87159)
For some reason in the version info panel, I don't have the navigation database option, 
![No Navigation Database Version](https://github.com/user-attachments/assets/f7c59c81-756e-4c96-a9be-e8c86a53bc86)
After the last couple of weeks of watching YouTube videos and reading numerous threads. I thought I would reach out and see if anyone in the community could help me,
I'm happy to obtain any additional images or resources if that would help identify the issue, 
Thank you in advance, 
Rhys

**Comment by williamsrh** [comment]
[MU1326-JTB-07403.12.1911710144.log](https://github.com/user-attachments/files/16532247/MU1326-JTB-07403.12.1911710144.log)
Adding the log file.

**Comment by n2ckjansens** [comment]
Same issue here. 

Tried to clean up Nav Db.
After cleaning DB MMI starts to see Map updates, but unfortunately after map update all symptoms returns :(

Found [here](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/448) that issue could be happen because of incorrect variantinfo. Tried different variantinfo's but nothing changed yet. 

Car: Audi A3 8V e-tron 2018.

**Comment by RawBee4572** [comment]
I had a similar problem, `variantinfo` was correct so playing with it didn't change anything.
what helped though was solution proposed here https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/448#discussioncomment-10648661 , more specifically removing all files in `gecache`, syncing and rebooting (in my specific case `truffles` directory was not present at all)

**Reply by williamsrh** [reply, marked as answer]
Thanks for the reply, that worked perfectly!

The way I ran the code was to add it to the individual.sh file (esd/scripts/individual) in M.I.B above the if statement (It runs automatically as soon as you open the folder): 

`echo "Fixing Maps Issue"`
`echo "Starting"`
`rm -Rf /net/mmx/mnt/gecache/*`
`rm -f /net/mmx/mnt/adb/truffles*`
`on -f mmx sync`
`echo "Finished"`


![image](https://github.com/user-attachments/assets/3d986ffa-80f8-4964-a13e-497e57716034)

To run the script, just go to: advanced_settings->Run_Individual_Script (Second Page)

I recommend formatting the database first: advanced_settings->format_nav_db

And then download the latest map pack for your unit from: https://mib-helper.com/show.php?all=maps

I hope this helps someone else who is having the same problem!

**Reply by RooleDj** [reply]
> Thanks for the reply, that worked perfectly!
> 
> The way I ran the code was to add it to the individual.sh file (esd/scripts/individual) in M.I.B above the if statement (It runs automatically as soon as you open the folder):
> 
> `echo "Fixing Maps Issue"` `echo "Starting"` `rm -Rf /net/mmx/mnt/gecache/*` `rm -f /net/mmx/mnt/adb/truffles*` `on -f mmx sync` `echo "Finished"`
> 
> ![image](https://private-user-images.githubusercontent.com/106061796/452396065-3d986ffa-80f8-4964-a13e-497e57716034.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDkzNzU0MTMsIm5iZiI6MTc0OTM3NTExMywicGF0aCI6Ii8xMDYwNjE3OTYvNDUyMzk2MDY1LTNkOTg2ZmZhLTgwZjgtNDk2NC1hMTNlLTQ5N2U1NzcxNjAzNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYwOFQwOTMxNTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMTNlNTEzZjUzZmViMDU0YjRhN2I5MzUwNTM4MDY5OTc1NTMzMjhhNTY5YjBiMjk3NGJlOWFiYmI3YzhhODRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.R6tOp2-bW0MgDF_fQOxueWnTA1fcjeptU_H8KegziU8)
> 
> To run the script, just go to: advanced_settings->Run_Individual_Script (Second Page)
> 
> I recommend formatting the database first: advanced_settings->format_nav_db
> 
> And then download the latest map pack for your unit from: https://mib-helper.com/show.php?all=maps
> 
> I hope this helps someone else who is having the same problem!

when you say advanced settings > run individual scripts, is that in MIB?
im having issues re-installing MIB (long story), i think its cached something somewhere from the previous install, im not sure, but its a new SD card which i formatted, and i leave it in the unit... i only had one originally so after install MIB i reformatted and put maps on it... so i have no backups etc lol so i uninstalled it

Do you think if i cleared all caches in one of the scripts that auto runs, prior to it reinstalling the launcher, this might help me install?
sorry to hijack, but i think it may be similar
